Info.plist.in

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrincipalClass</key>
+	<string>NSApplication</string>
```

Comment:
The important thing is those two lines.
Except this, the file is similar to MacOSXBundleInfo.plist.in Info.plist.in which is the default one used when no plist file is specified.
I don't remember the exact reason why this is needed, but it is something like a way for the application to tell it is "retina ready"